CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(aruco_catkin)
+
+find_package(catkin_simple REQUIRED)
+catkin_simple(ALL_DEPS_REQUIRED)
+
+include(ExternalProject)
+
+file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
+
+ExternalProject_Add(aruco_src
+  URL https://sourceforge.net/projects/aruco/files/3.0.0/aruco304.zip
+
+  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CATKIN_DEVEL_PREFIX}
+  BUILD_COMMAND CXXFLAGS=-i${CATKIN_DEVEL_PREFIX}/include make
```

Comment:
Why is the specification of the include path necessary? Actually not sure if this does not override the CXXFLAGS set by cmake. It would be nice if you wouldn't need to set this...

Reply:
It does build using `BUILD_COMMAND make` but I get these two warnings when building it:
`make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.`
`make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.`
How can I fix that?

Reply:
Since it's probably a warning that comes out of Aruco, you can leave it be as long  as it isn't an error.